plugins/importexport/medra/templates/crDepositErrors.tpl

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset={$defaultCharset|escape}" />
+		<style type="text/css">
+			table thead th {
+				border: 1px solid black; 
+				font-size: 18; 
+				font-weight: bold; 
+				padding: 10px;
+			}
+			table {
+				border-collapse: collapse;
+				font-size: 18;
+			}
+			table tbody td {
+				border: 1px solid black; 
+				font-size: 18; 
+				padding: 10px;
+			}
+			
+			table tbody tr:nth-child(even) {
+				background: #ABCDEF;
+			}
+			table tbody tr:nth-child(odd) {
+				background: #BDB9AA;
+			}
+			
+		</style>
+	</head>
+	<body>
+		<h2> {translate key="plugins.importexport.common.validationErrors"} </h2>
+		
+		<div> {translate key="plugins.importexport.medra.crossref.error.cause"} </div> <br/>
+		<div> {translate key="plugins.importexport.medra.crossref.error.number"}:  {$numberError} </div> <br/>
+		<div> {translate key="plugins.importexport.medra.crossref.error.details"}: </div> <br/>
+		<table>
+			<thead>
+				<tr>
+					<th>{translate key="plugins.importexport.medra.crossref.error.code"}</th>
+					<th>{translate key="plugins.importexport.medra.crossref.error.element"}</th>
+					<th>{translate key="plugins.importexport.medra.crossref.error.description"}</th>
+				</tr>
+				
+			</thead>
+			<tbody>
+				{foreach from=$headlines key=k item=v}
+					<tr>
+						<td>{$v['code']}</td>
+						<td>{$v['reference']}</td>
+						<td>{$v['description']}</td>
+					</tr>
+				{/foreach}
+			</tbody>
+		</table>
+		<h3>  {translate key="plugins.importexport.common.invalidXML"} </h3>
+		<pre>{$htmlspecialchars}</pre>
```

Comment:
I would recommend assigning `$xml` to the template manager directly, and using `{$xml|escape}` to display it. The variable name would be more meaningful, and escaping would be done in the template directly.

Reply:
In this case i used strip_unsafe_html to ensure that HTML elements could be display properly